### ISSUE TITLE
OnHandleRequest errors with redirect when using 2.0.5-pl

### ISSUE THREAD

**Issue by DESIGNfromWITHIN** [modxcms/Redirector#30]
Errors with redirect when using 2.0.5-pl
For my client site I get the following errors when I update redirector to 2.0.5-pl

This does not happen when I use 2.0.2-pl 

(I am sure it is related to: https://github.com/modxcms/Redirector/issues/24)

```
[2014-07-22 12:21:20] (ERROR @ /index.php) [OnHandleRequest]1
[2014-07-22 12:21:20] (ERROR @ /index.php) Error parsing condition with key 0: ('sdgsdg' REGEXP `modRedirect`.`pattern` OR 'sdgsdg' REGEXP CONCAT('^', `modRedirect`.`pattern`, '$'))
[2014-07-22 12:21:20] (ERROR @ /index.php) [OnHandleRequest]1
[2014-07-22 12:21:20] (ERROR @ /index.php) Error parsing condition with key 0: ('iphone/info6/' REGEXP `modRedirect`.`pattern` OR 'iphone/info6/' REGEXP CONCAT('^', `modRedirect`.`pattern`, '$'))
```

**Comment by pixelchutes** [comment]
:+1:

Error log has been littered with messages like this:

```
(ERROR @ /index.php) Error parsing condition with key 0: ('OLD_URL_HERE' REGEXP `modRedirect`.`pattern` OR 'OLD_URL_HERE' REGEXP CONCAT('^', `modRedirect`.`pattern`, '$'))
```

**Comment by pixelchutes** [comment]
Appears related to #22 (and #28)

**Comment by osabate** [comment]
Can we fix and close this bug? There are many discussions and attempts to fix this issue but still lies unresolved. Thanks!
[Here](https://github.com/modxcms/revolution/issues/11630), [here](http://forums.modx.com/thread/?thread=91434), [here](https://github.com/modxcms/Redirector/pull/28), ...

**Comment by bertoost** [comment]
Will be fixed in 2.0.6. Coming soon!
